srv/salt/ceph/stage/deploy/default.sls

```diff
@@ -61,13 +61,32 @@ admin:
     - tgt_type: compound
     - sls: ceph.admin
 
+mgr auth:
```

Comment:
Just had a timeout during deployment due to a, supposedly wrong ordering.

We try to auth against a monitor that is not yet deployed, see line 76.
This should be moved below the mon creation